src/components/Button.js

```diff
@@ -0,0 +1,24 @@
+import { NavLink } from 'react-router-dom';
+
+export function Button({
+  className = 'btn',
+  text = '',
+  href = '',
+  children,
+  onClick,
+}) {
+  return (
+    <>
+      {href !== '' ? (
+        <NavLink to={href} className={className} exact>
+          {children}
+          {text !== '' ? <span className="btn__text">{text}</span> : null}
+        </NavLink>
+      ) : (
+        <button className="btn" onClick={onClick}>
+          {children}
+        </button>
```

Comment:
Just a Q: regular buttons never have icons? Just the `NavLink`?

Reply:
Thanks, @laurenmbeatty! I changed a little this ^^. But, I'm honest. The `children` prop is helpful, also generic. You could pass anything 😄 . It was OK because it allowed us to pass an Icon component.

With the last changes, we assure you that both icon & Navlinks buttons have an icon.